exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/SerializationUtils.java

```diff
@@ -33,6 +36,16 @@ static void checkLength(byte[] array, int length) {
         "Expected an array of size %s, but was %s", length, array.length);
   }
 
+  /**
+   * Performs check that the array has no bytes after the position.
+   *
+   * @throws IllegalArgumentException thrown if any bytes in the tail exist
+   */
+  static void checkNoTailLeft(byte[] array, int pos) {
+    checkArgument(pos == array.length, "Expected no tail left in the array, but was %s",
```

Comment:
Probably, checkArrayLength would have been enough, but us we provide extra
information, the expected size (which `pos` actually is) must be included in
the error message.